python/ray/train/v2/_internal/execution/controller/controller.py

```diff
@@ -404,10 +419,17 @@ def _step(self) -> TrainControllerLoopIterationResult:
             scaling_decision = (
                 self._scaling_policy.make_decision_for_non_running_worker_group()
             )
+            if isinstance(scaling_decision, NoopDecision):
+                next_state = RestartingState()
+            elif isinstance(scaling_decision, ResizeDecision):
```

Comment:
Does this flow make sense? `Running(4 workers) -> Restarting -> Resizing(4 workers) -> Scheduling(4 workers)` 

Nothing is really being "resized" for the fixed # of workers fault tolerance case.